docs/ru/sql-reference/functions/other-functions.md

```diff
@@ -2088,3 +2088,54 @@ SELECT tcpPort();
 
 -   [tcp_port](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-tcp_port)
 
+## currentProfiles {#current-profiles}
+
+Возвращает список [профилей настроек](../../operations/access-rights.md#settings-profiles-management) для текущего пользователя. 
+
+Профиль настроек может быть указан при создании пользователя (см. [CREATE USER](../../sql-reference/statements/create/user.md#create-user-statement)). Позднее для изменения настроек могут применяться другие профили с помощью выражения [SET profile](../../sql-reference/statements/set.md#query-set). 
+
+**Синтаксис**
+
+``` sql
+currentProfiles()
+```
+
+**Возвращаемое значение**
+
+-   Список профилей настроек для текущего пользователя. 
+
+Тип: [Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md)).
+
+## enabledProfiles {#enabled-profiles}
+
+Возвращает профили настроек, назначенные пользователю как явно, так и неявно. Явно назначенные профили — это те же профили, которые возвращает функция [currentProfiles](#current-profiles). Неявно назначенные профили включают родительские профили других назначенных профилей; профили, назначенные с помощью предоставленных ролей; профили, назначенные с помощью собственных настроек; основной профиль по умолчанию (см. секцию `default_profile` в основном конфигурационном файле сервера).
+
+ Если какой-либо профиль встречается несколько раз (с некоторыми другими настройками между ними), последнее событие перезаписывает все предыдущие.
```

Comment:
```suggestion
Порядок профилей в возвращаемом массиве соответствует порядку, в котором эти профили были применены.
```

Reply:
Not sure again. I think it is a question of the duplicates are being removed.